ios/PurchasesHybridCommon/PurchasesHybridCommonTests/CustomerInfo+HybridAdditionsTests.swift

```diff
@@ -10,37 +10,54 @@ import Quick
 import Nimble
 @testable import PurchasesHybridCommon
 @testable import RevenueCat
+import Foundation
 
 class CustomerInfoHybridAdditionsTests: QuickSpec {
 
     override func spec() {
         describe("rc_dictionary") {
             context("managementURL") {
                 it("contains the management url when it exists") {
-                    let mockCustomerInfo = try CustomerInfo(data: [
-                        "request_date": "2019-08-16T10:30:42Z",
-                        "subscriber": [
-                            "first_seen": "2019-07-17T00:05:54Z",
-                            "management_url": "https://revenuecat.com",
-                            "original_app_user_id": "",
-                            "subscriptions": [:],
-                            "other_purchases": [:]
-                        ]])
+                    guard let mockCustomerInfo = CustomerInfo.fromJSON(
```

Comment:
I think `it` accepts throwing, so you could keep this as `throws`, and then the `try` will automatically fail the test.